yaml2sbml/yaml_schema.yaml

```diff
@@ -0,0 +1,188 @@
+# For syntax see: https://json-schema.org/understanding-json-schema/index.html
+#$schema: "https://json-schema.org/draft/2019-09/meta/core"
+$schema: "http://json-schema.org/draft-06/schema"
+description: yaml2sbml file format
+
+properties:
+
+  time:
+    items:
+      variable:
+        type: string
+        description: defines a time variable, in case the right hand side of the ODE is time-dependent.
+
+
+  parameters:
+    type: array
+    description: a list of the system parameters and respective definition.
+
+    items:
+      type: object
+      description: defines each system parameter.
+      properties:
+
+        parameterId:
+          type: string
+          description: any identifier which would be a valid identifier in SBML.
+        nominalValue:
+          type: number
+          description: the parameter's value. This is optional for SBML/PEtab generation, but will be needed
+            for model simulation.
+        parameterName:
+          type: string
+          description: the parameter name. This is optional.
+        parameterScale:
+          type: string
+          description: whether the parameter is in log (log10) or linear (lin) scale. This is optional.
```

Comment:
```suggestion
          description: whether the parameter is in `log`, `log10` or `lin` (i.e. linear)  scale. This is optional.
```

Reply:
(Since `log` (for base e) and `log10` (for base 10) are both valid parameter scales in PEtab)